src/main/java/ca/uhn/fhir/jpa/starter/controller/DashboardController.java

```diff
@@ -83,6 +82,28 @@ public ResponseEntity<?> getLastSyncTime(
 
 		return helperService.computeSyncTime(practitionerRoleId,env);
 	}
+	@RequestMapping(method = RequestMethod.GET, value = "/facilityLastSyncTime")
+	public Object getFacilityLastSyncTime(
+		@RequestHeader(name = "Authorization") String token,
+		@RequestParam("selectedOrganizationId") String selectedOrganizationId,
+		@RequestParam("env") String env){
+
+		String practitionerRoleId = Validation.getJWTToken(token).getPractitionerRoleId();
+		if (practitionerRoleId == null) {
+			return ResponseEntity.ok("Error : Practitioner Role Id not found in token");
+		}
+		HelperService.SyncTimeResponse syncTimeResponse = helperService.computeFacilitySyncTime(practitionerRoleId, env, selectedOrganizationId);
+		String eachClinicLastSyncTimes = syncTimeResponse.getEachClinicLastSyncTimes();
+		String formattedString1 = formatFacilitySyncTimeResponse(eachClinicLastSyncTimes);
```

Comment:
Change this variable name formattedString1. try avoiding numbers in variable name